src/reconcile/rec_col.c

```diff
@@ -765,9 +751,8 @@ __wt_rec_col_var(
                 ins = WT_SKIP_NEXT(ins);
             }
 
-            update_no_copy =
-              upd == NULL || !F_ISSET(upd, WT_UPDATE_RESTORED_FROM_DISK); /* No data copy */
-            repeat_count = 1;                                             /* Single record */
+            update_no_copy = upd == NULL; /* No data copy */
```

Comment:
I was considering removing this since the condition is not complex anymore. But I decided that it's nice that this has a name and signals intent.